provider/schema/resolvers_test.go

```diff
@@ -130,24 +131,74 @@ var UUIDTestTable = &Table{
 	},
 }
 
-func TestPathResolver(t *testing.T) {
-	r1 := PathResolver("Inner.Value")
-	r2 := PathResolver("Value")
-	r3 := PathResolver("unexported")
-	resource := NewResourceData(PostgresDialect{}, pathTestTable, nil, testStruct{Inner: innerStruct{Value: "bla"}, Value: 5, unexported: false}, nil, time.Now())
-	err := r1(context.TODO(), nil, resource, Column{Name: "test"})
-
-	assert.Nil(t, err)
-	assert.Equal(t, resource.Get("test"), "bla")
+func TestPathTableResolver(t *testing.T) {
+	results := make(chan interface{})
+	pathTableTests := []struct {
+		path          string
+		expectedValue interface{}
+	}{
+		{
+			path:          "Inner",
+			expectedValue: innerStruct{Value: "bla"},
+		},
+		{
+			path:          "Inner.Value",
+			expectedValue: "bla",
+		},
+		{
+			path:          "NilInner.Value",
+			expectedValue: "",
+		},
```

Comment:
Should we also add a case for the unexported value? What is expected to happen in that case?

Reply:
Added a test for that